pkg/config/setup/config.go

```diff
@@ -1648,6 +1649,17 @@ func podman(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("podman_db_path", "")
 }
 
+func infraAgent(config pkgconfigmodel.Setup) {
+	// Disables metric data submission (including Custom Metrics) so that hosts stop showing up in Datadog.
+	// Used namely for Error Tracking Standalone where Infrastructure Monitoring is not needed.
+	if enabled := os.Getenv("DD_INFRA_AGENT_ENABLE"); enabled == "false" {
```

Comment:
I may miss something but it is quite unusual to use the env variable. I would expect to have `config.GetBool("infra_agent")` instead.
Also the config option "infra_agent" is also not defined and so don't have a default value.